forge/db/models/User.js

```diff
@@ -99,16 +99,43 @@ module.exports = {
                 // determine if this user owns any teams
                 // throw an error if we would orphan any teams
                 const teams = await app.db.models.Team.forUser(user)
+                const teamBlockers = []
+                const teamOwnerships = []
```

Comment:
Nit pic suggestion: `ownedTeams` is more readable (sorry to be pedantic) (tis a suggestion only, take with salt ;) )

Reply:
I agree